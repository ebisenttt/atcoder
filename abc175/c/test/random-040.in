-444657243635948 490602834378380 16424807766894
