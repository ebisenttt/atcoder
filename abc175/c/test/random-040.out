15237373837084
